style.css

```diff
@@ -0,0 +1,492 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
+
+
+html {
+  font-size: 15px;
+  font-family: 'montserratregular';
+
+}
+
+body {
+  margin: 0 auto;
+  position: relative;
+  min-width: 320px;
+  max-width: 1440px;
+  
+}
+
+h1,
+h2 {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  font-family: 'montserratsemibold';
+  text-decoration: none;
+  color: black;
+}
+
+button {
+  border: none;
+  background: none;
+  cursor: pointer;
+  outline: none;
+  display: flex;
+  align-items: center;
+  padding: 0 40px;
+  color: black;
+  font-size: 13px; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)